README.md

```diff
@@ -37,22 +46,135 @@ api("com.github.spotify:openfeature-kotlin-sdk:[ANY_BRANCH]-SNAPSHOT")
 
 This will get a build from the head of the mentioned branch. 
 
-### Resolving a flag
+### Maven
+
+Installation via Maven Central is currently WIP
+
+## 🌟 Features
+
+- support for various backend [providers](https://openfeature.dev/docs/reference/concepts/provider)
+- easy integration and extension via [hooks](https://openfeature.dev/docs/reference/concepts/hooks)
+- bool, string, numeric, and object flag types
+- [context-aware](https://openfeature.dev/docs/reference/concepts/evaluation-context) evaluation
+
+## 🚀 Usage
+
+```kotlin
+    // configure a provider and get client
+    OpenFeatureAPI.setProvider(customProvider)
+    val client = OpenFeatureAPI.getClient()
+
+    // get a bool flag value
+    client.getBooleanValue("boolFlag", default = false)
+    
+    // get a bool flag value async
+    coroutineScope.launch {
+        WithContext(Dispatchers.IO) {
+            client.awaitProviderReady()
```

Comment:
Should we use the "raw" `eventObserver` here, and remove this helper function? It looks nice, but it's not in the OpenFeature Specs, and it adds more ways for users to wait on Provider's readiness, while it's probably nice to provide and document a single way to achieve this